internal/catalogmetadata/client/client.go

```diff
@@ -133,6 +135,7 @@ func (c *Client) doRequest(ctx context.Context, catalog *catalogd.ClusterCatalog
 
 	resp, err := client.Do(req)
 	if err != nil {
+		_ = httputil.LogCertificateVerificationError(err, ctrl.Log.WithName("catalog-client"))
```

Comment:
Why are we ignoring the return here
```
// This function unwraps the given error to find an CertificateVerificationError.
// It then logs the list of certificates found in the unwrapped error
// Returns:
// * true if a CertificateVerificationError is found
// * false if no CertificateVerificationError is found
```

Reply:
Because we don't need to log certs in `/etc/docker` here, because this is used for catalogd connections, not pulling images, The other places this is called, the return is used to determine if we want to look at `/etc/docker`

Reply:
I see, so we just called the function log error in to the logfile.